src/Microsoft.ML.StandardTrainers/Standard/LogisticRegression/MulticlassLogisticRegression.cs

```diff
@@ -979,40 +979,26 @@ private bool SaveAsOnnxCore(OnnxContext ctx, string[] outputs, string featureCol
         {
             Host.CheckValue(ctx, nameof(ctx));
 
-            string predictedLabelInt64 = null;
-            string predictedLabelUint32 = null;
-            // REVIEW: What is the right way to get the name of the predicted column?
-            for (int i = 0; i < outputs.Length; i++)
-            {
-                if (outputs[i] != DefaultColumnNames.PredictedLabel)
-                    continue;
-                predictedLabelUint32 = DefaultColumnNames.PredictedLabel;
-                predictedLabelInt64 = ctx.AddIntermediateVariable(NumberDataViewType.Int64, "PredictedLabelInt64", true);
-                outputs[i] = predictedLabelInt64;
-                break;
-            }
-
-            Host.CheckValue(predictedLabelInt64, nameof(predictedLabelInt64));
+            string labels = ctx.AddIntermediateVariable(NumberDataViewType.Int64, "Labels", true);
 
```

Comment:
Naming Nit: If this is not a vector, should this be in singluar as "Label" and "label"?